vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -108,7 +108,7 @@ struct ReactDevOptions {
   //! {EXTENSION} = ".bundle"
   //! Specify a value for a component, or leave empty to use the default.
   std::string SourceBundleHost; // Host domain (without port) for the bundler server. Default: "localhost".
-  std::string SourceBundlePort; // Host port for the bundler server. Default: "8081".
+  uint16_t SourceBundlePort{0}; // Host port for the bundler server. Default: "8081".
```

Comment:
>0 [](start = 28, length = 1)

comment says 8081 as the default, shouldn't we initialzie with 8081 then?

Reply:
-or- facebook::react::DevServerHelper::DefaultPackagerPort

---
In reply to: [445894069](https://github.com/microsoft/react-native-windows/pull/5344#discussion_r445894069) [](ancestors = 445894069)